gobblin-utility/src/main/java/org/apache/gobblin/util/filters/DotFileFilter.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.gobblin.util.filters;
+
+import org.apache.hadoop.fs.PathFilter;
+
+/**
+ * A {@link PathFilter} similar to {@link HiddenFilter}, but does not filter files starting with '_'.
+ */
+public class DotFileFilter extends HiddenFilter {
```

Comment:
What do you think is better - to have a separate class for dot, or a config setting, where users can override default ignored files in HiddenFilter?

I'm thinking about a couple use cases:
1. People want to copy everything, including hidden files
2. There are other types of files to ignore, like "~something"

Also, I found that we have RegexPathFilter, can users just use that to override filter behaviour?

Reply:
Yea I realized that too. closing it.